downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -72,6 +72,62 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
 - Introduced support for parsing XML with record types with default values as the expected type, using the default values where required (i.e., if a value corresponding to the record field is not present in the XML value).
 - Introduced the option to choose between semantic and syntactic equality of XML elements and attributes.
 
+#### `salesforce` package
+
+- Added support for Salesforce listener with Streaming API for real-time data synchronization and event-driven workflows.
+- Added support for base types with `salesforce.types` submodule.
+- Reorganized salesforce clients for better usability.
+
+### New connectors
```

Comment:
@niveathika IMO we can remove this sub section and include the same as the notes under the particular connectors. We've followed the same release notes format in U9 as well https://ballerina.io/downloads/swan-lake-release-notes/swan-lake-2201.9.0#revamped-connectors.

WDYT?

Reply:
I am -1 to call these as revamped connectors, as these functionalities were not available previously in ballerina.

Reply:
@niveathika This structure was proposed in Update 9 to capture all the changes related to the connector revamping effort(where these new connector were also a part of the same effort). But I understand your concern on having `Revamped Connectors` as the title for all the changes. In that case we can rename it as `Connector updates` and keep all the new/revamped connector release under that single section. wdyt?

Reply:
+1 for that